release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -123,6 +139,24 @@ console. This lets you view metrics that bring transparency to the
 [id="ocp-4-4-web-console"]
 === Web console
 
+[id="ocp-4-4-marketplace-integration-in-operator-hub"]
+==== IBM Marketplace integration in OperatorHub
+
+IBM Marketplace is now integrated in the OperatorHub, which is located in the
```

Comment:
"integrated in" or "integrated with?"

Is there a clearer word than "integrated?"